facebook/delphiFacebook/R/contingency_variables.R

```diff
@@ -227,6 +227,45 @@ code_health <- function(input_data, wave) {
   return(input_data)
 }
 
+#' Vaccinated summary variable for making table cuts
+#'
+#' @param input_data input data frame of raw survey data
+#' @param wave integer indicating survey version
+#' 
+#' @return augmented data frame
+code_vaccinated_breakdown <- function(input_data, wave) {
+  # grouping variable - vaccination status
+  if (any(c("V1", "V3", "V3a", "V11a") %in% names(input_data))){
```

Comment:
Is this outer `if` really necessary? Feels like you could just have the inner `if` and `else if`, and then you only need one `else` clause.